client/src/modules/stock/dashboard/dashboard.js

```diff
@@ -0,0 +1,20 @@
+angular.module('bhima.controllers')
+  .controller('StockDashBoardController', StockDashBoardController);
+
+StockDashBoardController.$inject = [
+];
+
+/**
+ * @function StockDashBoardController
+ *
+ * @description
+ * This controller binds the basic CRUD operations on the enterprise.
+ */
+function StockDashBoardController() {
+  // fired on startup
+  function startup() {
+
+  }
```

Comment:
?

Reply:
See that everything is done with the component, there is unfortunately nothing inside the model